src/app/components/common/TagFilter.tsx

```
@@ -0,0 +1,76 @@
+          value={tag}
+          checked={checked}
+          onChange={onChange}
+          className="sr-only"
+          aria-label={tag}
+        />
+        <span
+          className="h-7 flex items-center justify-center"
+          aria-hidden="true"
+        >
+          {checked ? (
+            <svg width="18" height="18" viewBox="0 0 18 18" fill="none">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)